pkg/bnd/verifier.go

```diff
@@ -6,6 +6,7 @@ package bnd
 import (
 	"fmt"
 
+	"github.com/sigstore/sigstore-go/pkg/bundle"
```

Comment:
Maybe I shouldn't be importing this and should instead add the functionality needed to `v.bundleVerifier`?  Maybe `v.bundleVerifier.OpenInlineBundle(...)`?

Reply:
Ideally as we can mock the internal implementation, but it is just one function. I'll do it in a follow up, no worries :)